src/gettingstarted/template/next-steps.md

```diff
@@ -0,0 +1,86 @@
+# Start with a template
+
+## Next steps
+
+In this guide you created a project using the [management console](/administration/web.md) and installed the [Platform.sh CLI](/development/cli.md).
+
+Don't stop now! There are far more features that make Platform.sh profoundly helpful to developers that you have left to explore.
+
+### Import your own code
```

Comment:
I feel like these should be level 2 headers, not level 3.  They don't really need to be sub-items of "Next steps".